packages/hash/api/src/graphql/resolvers/entityType/createEntityType.ts

```diff
@@ -18,7 +18,7 @@ export const deprecatedCreateEntityType: ResolverFn<
   dataSources.db.transaction(async (client) => {
     const entityType = await EntityType.create(client, {
       accountId,
-      createdByAccountId: user.accountId,
+      createdByAccountId: user.entityId,
```

Comment:
The reason I've had to touch the legacy GQL resolvers in this PR is because we are attaching the updated `UserModel` class to the GQL context, so changes to its type propagate into these files. These uses of `user.accountId` were incorrect to begin with, so have replaced them with `user.entityId` which is closer to what we will end up with.